src/EventListener/AdministratorExportHistoryListener.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use AppBundle\Entity\Administrator;
+use AppBundle\Entity\Reporting\AdministratorExportHistory;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Security\Core\Security;
+
+class AdministratorExportHistoryListener implements EventSubscriberInterface
+{
+    private $security;
+    private $entityManager;
+
+    public function __construct(Security $security, EntityManagerInterface $entityManager)
+    {
+        $this->security = $security;
+        $this->entityManager = $entityManager;
+    }
+
+    public function onKernelResponse(FilterResponseEvent $event)
+    {
+        $administrator = $this->security->getUser();
+
+        if (!$administrator instanceof Administrator) {
+            return;
+        }
+
+        $request = $event->getRequest();
+        $routeName = $request->get('_route');
+
+        if (!preg_match('/^admin_(.)+_export$/', $routeName)) {
```

Comment:
what about checking 200 response status code too?

Reply:
why ?
i want to know if someone tried to export things.
i dont care if it succeeded or not?

Reply:
yes.
the reporting is about the export requests. not the export responses